upgrade/upgrade-scripts/verify-impl

```diff
@@ -36,6 +37,94 @@ function run_support_info() {
 	return "$rc"
 }
 
+function verify_run_verify_jar() {
+	echo "Verify - Run verify jar."
+
+	/usr/bin/java \
+		-Dlog.dir=/var/delphix/server/upgrade-verify \
+		-Dmdsverify=true \
+		"${VERIFY_DEBUG_OPT}" \
+		-jar /opt/delphix/server/lib/exec/upgrade-verify/upgrade-verify.jar \
+		-d "${opt_o:-${LOG_DIRECTORY}/${CONTAINER}/upgrade_verify.json}" \
+		-f "${opt_f:-1}" \
+		-l "${opt_l:-en-US}" \
+		-v "$IMAGE_VERSION" \
+		-pl 60 -ph 80 \
+		"${VERIFY_LIVE_MDS_OPT}" ||
+		die "'upgrade-verify.jar' failed in verification container"
+
+	#
+	# This name is used by the "upgrade-verify.jar" just executed, so we
+	# cannot change this value without also modifying that JAR.
+	#
+	MDS_SNAPNAME="MDS-CLONE-upgradeverify"
+
+	# TODO : Make sure MDS clone gets cleaned up if verification fails.
```

Comment:
TODO

Reply:
I think we can accomplish the cleanup on failure using another "cleanup" function with `trap`.. e.g.

```
function verify_run_verify_jar_cleanup() {
    if clone_is_running; then
        /opt/delphix/server/bin/dx_manage_pg stop ...
    fi
    if clone_exists; then
        /opt/delphix/server/bin/dx_manage_pg cleanup ...
    fi
    /opt/delphix/server/bin/support_info ...
}

function verify_run_verify_jar {
    trap verify_run_verify_jar_cleanup EXIT
    ... # run JAR
    /opt/delphix/server/bin/dx_manage_pg stop ...
    /opt/delphix/server/bin/dx_manage_pg cleanup ...
    trap - EXIT
}
```

we'd then need similar "cleanup" logic for the stack startup function:
```
function verify_stack_startup_cleanup {
    if ...; then
        /opt/delphix/server/bin/dx_manage_pg stop ...
    fi
    if ...; then
        /opt/delphix/server/bin/dx_manage_pg cleanup ...
    fi
    /opt/delphix/server/bin/support_info ...
}

function verify_stack_startup {
    trap verify_stack_startup_cleanup EXIT
    ... # run stack startup logic
    /opt/delphix/server/bin/dx_manage_pg stop ...
    /opt/delphix/server/bin/dx_manage_pg cleanup ...
    trap - EXIT
}
```

And then we'd probably also remove the line:
```
trap run_support_info EXIT
```
and the `run_support_info` function since we'd be running the support info script from the other "cleanup" functions.